### PR TITLE
Polish the limits section

### PR DESCRIPTION
**Pull request by certik**
Now it forms a readable text. Two examples using SymPy were added.

### CODE REVIEW

File: gruntz.tex [hunk 1]
```diff
@@ -1,55 +1,98 @@
-We first define comparability classes by calculating $L$:
+SymPy calculates limits using the Gruntz algorithm, as described in
+\cite{Gruntz1996limits}. The basic idea is as follows: any limit can be
```
Comment (asmeurer) [inline gruntz.tex]: Should be `~\cite`

File: gruntz.tex [hunk 2]
```diff
@@ -1,55 +1,98 @@
...
+converted to a limit $\lim_{x\to\infty} f(x)$ by substitutions like
+$x\to{1\over x}$.
+Then we identify in $f(x)$ the most varying subexpression $\omega$ (that
```
Comment (asmeurer) [inline gruntz.tex]: Don't use the first person

Comment (certik) [inline gruntz.tex]: Do you want to use the passive form?

Comment (asmeurer) [inline gruntz.tex]: The previous sentence is already passive, so I think this one should match.

File: gruntz.tex (hunk 1 above)
Comment (certik) [inline gruntz.tex]: Fixed in 34f775b1060a6b1e714bee3a713cb6434b59b0aa.

File: gruntz.tex [hunk 3]
```diff
@@ -1,55 +1,98 @@
...
+expand $f(x)$ into a series with respect to $\omega$. Any positive powers of
+$\omega$ converge to zero. If there are negative powers of $\omega$, then the
+limit is infinite. The constant term (independent of $\omega$, but could depend
+on $x$) then determines the limit (one might need to recursively apply the
+Gruntz algorithm on this term to determine the limit).
+
+To determine the most varying subexpression, we first need to define
+comparability classes, by calculating $L$:
 \begin{equation}
 L\equiv \lim_{x\to\infty} {\log |f(x)| \over \log |g(x)|}
```
Comment (asmeurer) [inline gruntz.tex:15]: Why is this \equiv and not just =?

Comment (certik) [inline gruntz.tex:15]: Since I am defining `L`, I used `≡` (`\equiv`).

**Comment by asmeurer** [comment]
Above in the calculus section I used `\lim\limits_{x \to \infty}`, because the default inline limit looks really bad to me. What do you think about that? We should at least be consistent with it.

File: gruntz.tex [hunk 4]
```diff
@@ -1,55 +1,98 @@
...
-$\omega\to0$:
-$$f(x) = {1\over x}-{1\over\omega}+{1\over\omega}e^{2\omega}
+Let us now illustrate the Gruntz algorithm on the following example:
+\begin{equation}
+    \label{gruntz_example_fn}
+f(x) = e^{x+2e^{-x}} - e^x + {1\over x} \,.
+\end{equation}
+We want to calculate $\lim_{x\to\infty} f(x)$.
+We first determine the set of most rapidly varying subexpressions, the so
+called \textit{mrv set}, for \eqref{gruntz_example_fn} we obtain
```
Comment (asmeurer) [inline gruntz.tex]: Replace comma with period

Comment (asmeurer) [inline gruntz.tex]: And comma after the eqref

File: gruntz.tex (hunk 2 above)
Comment (certik) [inline gruntz.tex]: Ok, I converted the whole PR to passive form in a3226e6f8fcb8c6ef3a8d7afefa3f51018659542.

File: gruntz.tex (hunk 4 above)
Comment (certik) [inline gruntz.tex]: Fixed in 886ee4c9746695b60c7531bd6187003f80a7f79b.

File: gruntz.tex [hunk 5]
```diff
@@ -1,55 +1,94 @@
...
+
+\begin{verbatim}
+>>> from sympy.series.gruntz import mrv
+>>> mrv(exp(x+2*exp(-x))-exp(x) + 1/x, x)[0].keys()
+dict_keys([exp(x + 2*exp(-x)), exp(x), exp(-x)])
+\end{verbatim}
+
+Next any item $\omega$ is taken from mrv that converges to zero for
+$x\to\infty$. The item $\omega=e^{-x}$ is obtained. If such a term is not
+present in the mrv set (i.e. all terms converge to infinity instead of zero),
```
Comment (asmeurer) [inline gruntz.tex]: comma after `i.e.`

File: gruntz.tex [hunk 6]
```diff
@@ -1,55 +1,94 @@
...
+substituted back into $f(x)$ and expanded with respect to $\omega$:
+\begin{equation}
+    \label{gruntz_example_fn2}
+f(x) = {1\over x}-{1\over\omega}+{1\over\omega}e^{2\omega}
+     = 2+{1\over x} + 2\omega + O(\omega^2)
+\end{equation}
+
+Since $\omega$ is from the mrv set, then in the limit $x\to\infty$ it is
+$\omega\to0$ and so $2\omega + O(\omega^2) \to 0$ in
+\eqref{gruntz_example_fn2}:
```
Comment (asmeurer) [inline gruntz.tex]: ~\eqref

**Comment by asmeurer** [comment]
Other than those two comments (based on `chktex`), I'm OK to merge, unless someone else wants to review this as well. Some more fixes may come in when we do the full read-through, but that will come later.

**Comment by certik** [comment]
> Above in the calculus section I used `\lim\limits_{x \to \infty}`, because the default inline limit looks really bad to me. What do you think about that? We should at least be consistent with it. 

Yeah, it looks really crappy. I converted to your way in e8ee891aabc321f6f0913dac4a151f79b51aaf56.

**Comment by asmeurer** [comment]
My way also looks crappy, in the sense that it adds extra space between the lines. So you have to balance that.

File: gruntz.tex (hunk 5 above)
Comment (certik) [inline gruntz.tex]: Fixed in 365376bad7daee059d9dcaf1805c8864118ce79c.

File: gruntz.tex (hunk 6 above)
Comment (certik) [inline gruntz.tex]: Fixed in 66ec1f81e0cae62c8470650b50212e19ca97b41c.

**Comment by certik** [comment]
Yeah, the extra space between lines also sucks, but I think it looks a bit better.

**Comment by certik** [comment]
Thanks for the review @asmeurer. I fixed all comments. This is ready to go in once tests pass.

**Pull request merged**